### ISSUE TITLE
AOS Animated Element Position Offset

### ISSUE THREAD

**Issue by sanjay-lunay** [michalsnik/aos#700]
## This is:
- Question

## Specifications

  - AOS version: Next 
  - OS: Windows
  - Browser: Chrome

## Expected Behavior

Text element should animate with fade-left in to the position defined via CSS.

## Actual Behavior

Instead, it is offset from this position by quite a bit on the Y-axis. 

## Steps to Reproduce the Problem

1. In CSS define position as aboslute for element
2. Apply AOS library via CDN stylesheet and script and intiate it 
3. Add fade left tag to any text (or element)

## Detailed Description

I would like to know if others have experienced this same problem and if they have found a fix to it.

**Comment by sanjay-lunay** [comment]
![good](https://user-images.githubusercontent.com/87889605/126858664-7bfc6b2b-717e-4784-ac17-c57bb8a53b7b.png)
![error](https://user-images.githubusercontent.com/87889605/126858665-2d3849f6-b293-497f-82b1-ef3ea07807a8.png)

this text i have is centered via CSS, but when I add the AOS animation, it is no longer in that position. This is that same for all other text elements with position absolute. However, when I move full background images, the problem no longer seems to persist.

**Comment by Verrok** [comment]
same

**Comment by sanjay-lunay** [comment]
> same

Hey! Did you manage to get it working? If not, did you find some other alternative?

**Comment by Verrok** [comment]
> > same
> 
> Hey! Did you manage to get it working? If not, did you find some other alternative?

nah, switched to Anime.js + triggering via ScrollMagic, works perfectly

**Comment by sanjay-lunay** [comment]
> > > same
> > 
> > 
> > Hey! Did you manage to get it working? If not, did you find some other alternative?
> 
> nah, switched to Anime.js + triggering via ScrollMagic, works perfectly

okay, thanks so much!
